### ISSUE TITLE
Creating an About Us Page for Fodome

### ISSUE THREAD

**Issue by MohamedMafaz** [MohamedMafaz/Fodome#7]
We need to create an "About Us" page for Fodome that provides information about our company, our mission, and our team. The UI for the page will be provided, so you just need to focus on the development part.

If you're interested in working on this issue, please leave a comment to let us know that you're working on it. This will help us avoid having multiple contributors working on the same issue at the same time.

We welcome everyone to contribute to this issue, even if you have not been assigned to it. So, if you're interested, feel free to start working on it and let us know if you have any questions or need any help.

**Comment by willdelorm** [comment]
Hi @MohamedMafaz, I'd like to build this page for you too!

**Comment by MohamedMafaz** [comment]
Hi Willdelorm,
First try to build Contact Us page. After that I'll give you the UI for About Us page

**Comment by Shaju06** [comment]
Hello, can I work on this issue if no one is working?

**Comment by MohamedMafaz** [comment]
https://github.com/MohamedMafaz/Fodome/issues/4   Can you build Why Fodome? Page

**Comment by Shaju06** [comment]
Sure, please provide design for it and just to confirm it's plain html css right?

**Comment by MohamedMafaz** [comment]
https://github.com/MohamedMafaz/Fodome/files/11428153/Fodome.UI.zip  Yeah! Plain css and html. But, It must be responsive

**Comment by Shaju06** [comment]
Unable to open this .fig in figma

**Comment by Shaju06** [comment]
Nvm just figured it out, can you assign me the same issue?

**Comment by MohamedMafaz** [comment]
Hey @Shaju06 
Do you know how to deploy a dynamic website?

**Comment by MohamedMafaz** [comment]
Hey @Shaju06 
Do you know how to deploy a dynamic website?

**Comment by MohamedMafaz** [comment]
Hey @Shaju06 
Do you know how to deploy a dynamic website?

**Comment by Shaju06** [comment]
You mean deploy this website on server right?

**Comment by MohamedMafaz** [comment]
Yeah! But need to deploy it in any free hosting provider like PythonAnywhere

**Comment by MohamedMafaz** [comment]
Hey @willdelorm 
Are you working on this issue? If yes pls comment below.

**Comment by barry-g1076** [comment]
> Yeah! But need to deploy it in any free hosting provider like PythonAnywhere

you can use this website for hosting https://www.000webhost.com/

**Comment by willdelorm** [comment]
> 

Hey Mohamed, I didn't get the go-ahead to build this page.

**Comment by MohamedMafaz** [comment]
Okay bro! Thanks for your involvement. You already build contact us page for us which is still stand alone. Thank you once again 🙂

**Comment by Shahbaz898414** [comment]
@MohamedMafaz 
this issue is still open?
